tools/experiment.py

```diff
@@ -347,10 +370,16 @@ def _parse_train_record(self, file_path: Path):
         for line in lines:
             iter_history = json.loads(line)
             if iter_history.get("mode") == "train":
-                self._iter_time_arr.append(iter_history["time"])
-                self._data_time_arr.append(iter_history["data_time"])
                 if iter_history["epoch"] > last_epoch:
                     last_epoch = iter_history["epoch"]
+                if last_epoch <= 2:  # if epoch >= 2, first epcoh is excluded from the calcuation
+                    iter_time = []
+                    data_time = []
+                iter_time.append(iter_history["time"])
+                data_time.append(iter_history["data_time"])
+
+        self._iter_time_arr.extend(iter_time)
+        self._data_time_arr.extend(data_time)
```

Comment:
There is a potential risk of `iter_time` and `data_time` not defined. Why don't you define those empty lists just before the loop?

Reply:
applied at https://github.com/openvinotoolkit/training_extensions/pull/2757/commits/c5736fdf23a69690942c3b0c658901bb753a532a